aemedge/blocks/accordion/accordion.css

```diff
@@ -1,57 +1,48 @@
-.accordion details {
-  border: 1px solid var(--dark-color);
+.accordion p {
+  margin: 0
 }
 
-/* stylelint-disable-next-line no-descending-specificity */
-.accordion details + details {
-  margin-top: 16px;
+.accordion details {
+  border-bottom: 1px solid var(--udexColorGrey3);
+  width: 100%
 }
 
 .accordion details summary {
   position: relative;
-  padding: 0 16px;
-  padding-right: 48px;
+  padding: var(--udexSpacer12) var(--udexSpacer12) var(--udexSpacer12) var(--udexSpacer8);
   cursor: pointer;
   list-style: none;
   overflow: auto;
-  transition: background-color 0.2s;
+  display: inline-flex;
+  align-items: center;
+  font-size: var(--udexTypographyHeadingMediumXXXSFontSize);
+  font-weight: var(--udexTypographyFontWeightMedium);
+  box-sizing: border-box;
+  width: 100%;
 }
 
-.accordion details[open] summary {
-  background-color: var(--light-color);
-}
+.accordion .accordion-arrow {
+  box-sizing: border-box;
+  display: inline-block;
 
-.accordion details summary:focus,
-.accordion details summary:hover {
-  background-color: var(--dark-color);
-}
+  /* Helps ensure icon and text are aligned properly */
+  line-height: var(--udexTypographyHeadingMediumXXXSFontSize);
+  width: 42px;
+  height: 42px;
 
-.accordion details summary::-webkit-details-marker {
-  display: none;
+  /* Icon image should be 18px x 18px */
+  padding: calc((42px - 18px) / 2);
+  margin-right: var(--udexSpacer12);
 }
 
-.accordion details summary::after {
-  content: "";
-  position: absolute;
-  top: 50%;
-  right: 18px;
-  transform: translateY(-50%) rotate(135deg);
-  width: 9px;
-  height: 9px;
-  border: 2px solid;
-  border-width: 2px 2px 0 0;
-  transition: transform 0.2s;
+.accordion details[open] .accordion-arrow {
```

Comment:
Can this be changed from `open` to `aria-expanded`?

Reply:
`details` elements seem to have an implicit aria role of "group", which isn't included in the list of [roles](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded#associated_roles) that use `aria-expanded`. I guess we should stick with `open`.